src/Components/List/List.jsx

```diff
@@ -38,18 +68,58 @@ const purchaseItem = (item, token) => {
         lastPurchased: new Date(),
         oldPurchased: item.lastPurchased,
         numberOfPurchases: numberOfPurchases,
-        calculatedEstimate: calculateEstimate(
-          item.frequency,
-          lastInterval,
-          numberOfPurchases,
-        ),
+        calculatedEstimate: calculatedEstimate,
       },
     });
 };
 
+//Color code based on days until next purchase or inactivity with corresponding aria label
+const colorCode = (item) => {
+  const daysSincePurchased = dayjs().diff(
+    getLastPurchaseDate(item, item.lastPurchased),
+    'day',
+  );
+  const estimatedCountdown = item.calculatedEstimate - daysSincePurchased;
+  const elapsedTime = dayjs().isSameOrAfter(
+    dayjs(getLastPurchaseDate(item, item.lastPurchased)).add(
+      item.calculatedEstimate * 2,
+      'day',
+    ),
+  );
+
+  if (elapsedTime === true) {
+    return ['inactive', 'Inactive'];
```

Comment:
I ran into a case where items that are purchased once are considered inactive right away. It might have to do with the estimate function setting the calculatedEstimate of items that are purchased once to 0, which is throwing some things off.

Reply:
We did think this was a bit odd too, but the description for the issue said to make single purchased items inactive. I guess we're assuming one-off purchases are never going to be bought again until they actually are bought again. It's part of why we need to forcibly move inactive to the bottom because 0 calculatedEstimate sits at the top, but inactive that hasn't been bought in awhile will naturally go to the bottom.

**Issue Description:**
_"Inactive (when there’s only 1 purchase in the database or the purchase is really out of date [the time that has elapsed since the last purchase is 2x what was estimated])"_

Reply:
Ah! I must have missed that in the issue. If it's an intended effect then it should be fine.